scripts/qa_crowbarsetup.sh

```diff
@@ -4169,9 +4176,26 @@ puts y.to_yaml" > /root/keystone-test-pw-reset.yaml
     safely crowbar batch --timeout 900 build < /root/keystone-test-pw-reset.yaml
 }
 
+function onadmin_setupsalt
+{
+    test -e /opt/dell/crowbar_framework/barclamps/salt.yml || return 0
+    add_salt_repo
+    do_one_proposal salt default
+}
+
+function onadmin_testsalt
+{
+    test -e /opt/dell/crowbar_framework/barclamps/salt.yml || return 0
+    test -e /etc/salt/roster || onadmin_setupsalt
+    salt-ssh \* test.ping | tee /tmp/salt.test.log
+    # compare successful pings to $nodenumber (+1 for crowbar)
+    test $((1 + nodenumber)) = $(grep "^    True$" /tmp/salt.test.log | wc -l)
```

Comment:
Could you write this regexp as `^.*True` so that we don't rely on a fixed number of whitespaces?

Reply:
now it is 'one or more' spaces